src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -180,7 +180,11 @@ static DataTestUtility()
             AliasName = c.AliasName;
             IsJsonSupported = c.IsJsonSupported;
 
+#if NET9_0_OR_GREATER
+//TODO: set protocol here?
```

Comment:
This is an outstanding question. If tests pass without setting a default protocol, nothing to do here.